src/Rector/AbstractRector.php

```diff
@@ -172,7 +170,6 @@ public function autowireAbstractRector(
         $this->nodeNameResolver = $nodeNameResolver;
         $this->nodeTypeResolver = $nodeTypeResolver;
         $this->simpleCallableNodeTraverser = $simpleCallableNodeTraverser;
-        $this->visibilityManipulator = $visibilityManipulator;
```

Comment:
It should be also removed from the parameters

Reply:
I wonder why Rector rule missed it :thinking: :) 

Thanks :+1: We'll add a rule to cover this too

Reply:
I made issue for it https://github.com/rectorphp/rector/issues/6830